src/style.css

```diff
@@ -1,3 +1,76 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  padding-left: 50px;
+  padding-right: 50px;
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+}
+
+li {
+  display: flex;
+  height: 60px;
+  position: relative;
+  margin-right: 20px;
+}
+
+li:last-child {
```

Comment:
You still style tags, it's a bad practice and an idea. The same thing with the links below.
It looks ok when you have such a small app like this, but it's not. For tags you can only reset or normalize styles so they will be the same in all browsers. But when you set specific styles to them, you lose flexibility and have problems with unexpected behavior when you add another list that should have different styles.

Reply:
Hi
Im sorry for taking your time
There was problem with my commits, as you can see in index html was styles that not exist in styles.css file 
I missed `styles.scc` in my commits